### ISSUE TITLE
Pulmonary cavity

### ISSUE THREAD

**Issue by katrinknoflach** [obophenotype/human-phenotype-ontology#6704]
**Preferred term label:**
Pulmonary cavity

**Synonyms**
Pulmonary cavern

**Definition (free text, please give PubMed ID)**
A gas-filled space, seen as lucency or low-attenuation area, within a nodule, mass or area of parenchymal consolidations. It has a clearly defined wall > 4 mm thick.

PMID: 18195376
PMID: 29518379

**Comment**
A pulmonary cavity can be caused by microorganisms (bacteria including mycobacteria, funghi, viruses, parasites), malignant processes or autoimmune disorders.

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary thoracic imaging finding

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by katrinknoflach** [comment]
@mgriese
@crapp

**Comment by pnrobinson** [comment]
?? What relation should this have to https://hpo.jax.org/app/browse/term/HP:0032445

**Comment by pnrobinson** [comment]
added

**Comment by mgriese** [comment]
Cavitary masses or cavitary nodules are differentiated from cysts, as the former are generated as holes in consolidations/Masses, nodules, and the latter are generated from vaning tissue. So would not use synonym.
